timers.js

```diff
@@ -1,4 +1,15 @@
-
-
-
-
+$("#start").on("click",function updateTime(){
+  var time = 0;
+  var count = setInterval(function(){
+    time= time + 1;
+    console.log(time);
+    $("#timer").text(time + " s");
+  },1000)
+  $("#pause").on("click",function pauseClock(){
```

Comment:
this works! but will set up an event listener for pause every time you click start